apps/examples/linux_rpc_demo/linux-rpmsg-rpc-demo.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef RPMSG_RPC_DEMO_H
+#define RPMSG_RPC_DEMO_H
+
+#define RPMSG_SERVICE_NAME         "rpmsg-openamp-demo-channel"
```

Comment:
The name service should be defined inrpmsg_rpc_client_server.h with following define

#define RPMSG_RPC_SERVICE_NAME "rpmsg-rpc"